src/Modix.Bot/Responders/StarboardReactionResponder.cs

```diff
@@ -34,13 +35,21 @@ private async Task HandleReactionAsync(Cacheable<IUserMessage, ulong> cachedMess
                 return;
             }
 
+            if (channel.ChannelType == ChannelType.PrivateThread)
+            {
+                return;
+            }
+
             var isIgnoredFromStarboard = await designatedChannelService
                 .ChannelHasDesignation(channel.Guild.Id, channel.Id, DesignatedChannelType.IgnoredFromStarboard, default);
 
+            var isParentIgnoredFromStarboard = channel is SocketThreadChannel { ParentChannel: not null } threadChannel && await designatedChannelService
+                .ChannelHasDesignation(channel.Guild.Id, threadChannel.ParentChannel.Id, DesignatedChannelType.IgnoredFromStarboard, default);
```

Comment:
I would like to isolate this into a method, such that it returns if the channel is ignored, not captured in two variables.

```cs
private async Task<bool> IsChannelIgnoredFromStarboard(IGuildChannel channel)
{
	var hasDirectDesignation = await designatedChannelService
                .ChannelHasDesignation(channel.Guild.Id, channel.Id, DesignatedChannelType.IgnoredFromStarboard, default);

	if(hasDirectDesignation)
		return true;

	if(channel is SocketThreadChannel { ParentChannel: not null } threadChannel)
	{
		var parentChannelId = threadChannel.ParentChannel.Id;
		return await designatedChannelService
                .ChannelHasDesignation(channel.Guild.Id, parentChannelId, DesignatedChannelType.IgnoredFromStarboard, default);
	}

	return false;
}
```